server/util/src/constants.rs

```diff
@@ -5,7 +5,7 @@ pub const INVENTORY_ADJUSTMENT_NAME_CODE: &str = "invad";
 /// Code for the special repack name
 pub const REPACK_NAME_CODE: &str = "repack";
 /// Number of days in a month (used in AMC calculation)
-pub const NUMBER_OF_DAYS_IN_A_MONTH: f64 = 30.0;
+pub const NUMBER_OF_DAYS_IN_A_MONTH: f64 = 365.25 / 12.0;
```

Comment:
This constant is already defined in this file as:

```rs
/// Use this value for accurate average month length (i.e. when considered over
/// a long time period)
pub const DAYS_PER_MONTH: f64 = 365.25 / 12.0;
```

Can we combine them now, since they're the same value now?

Reply:
[b70b580](https://github.com/msupply-foundation/open-msupply/pull/6042/commits/b70b5808cd79a9affb56f579ec35f0f9eb4f742d)